src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random number = new Random();
+    private final ColorSupplier color = new ColorSupplier();
+    private final int range = 100;
```

Comment:
It is better to use the constant
```suggestion
    private static final int RANGE = 100;
```